tox.ini

```diff
@@ -60,6 +60,9 @@ commands =
 
 [testenv:unit]
 description = Run unit tests
+set_env =
+    {[testenv]set_env}
+    LIBJUJU_VERSION_SPECIFIER = {env:LIBJUJU_VERSION_SPECIFIER:2.9.44.1}
```

Comment:
Should it be Juju3?

(If yes, then we should adjust the integration tests as well... Meaning that `poetry.lock` should pin down libjuju version to 3, and we should downgrade in `ci.yaml` if `LIBJUJU_VERSION_SPECIFIER` refers to Juju2._

Reply:
@paulomach @taurus-forever @shayancanonical @carlcsaposs-canonical 

Guys, what do you think about which Juju version to consider as a default?

Reply:
Changed it to Juju3 according to https://chat.canonical.com/canonical/pl/ixcqfiuok7gozqni6ikotu5uiy